source/get-started/setup/index.md

```diff
@@ -3,186 +3,116 @@ title: Setting Up PatternFly
-            <code>&lt;link href="bower_components/patternfly/dist/css/patternfly-additions.min.css" rel="stylesheet" media="screen, print"&gt;</code>
-          </pre>
-        <p>
-          Add the following script references to your HTML file. If you would like additional pieces, you can find them in the <a href="https://github.com/patternfly/patternfly/blob/master/QUICKSTART.md">PatternFly Quickstart Guide</a>.
-        </p>
-        <p>
-          <pre>
-            <code>&lt;!-- jQuery --&gt;</code>
-            <code>&lt;script src="bower_components/jquery/dist/jquery.min.js"&gt;&lt;/script&gt;</code>
-            <code>&lt;!-- Bootstrap --&gt;</code>
-            <code>&lt;script src="bower_components/bootstrap/dist/js/bootstrap.min.js"&gt;&lt;/script&gt;</code>
-            <code>&lt;!-- PatternFly --&gt;</code>
-            <code>&lt;script src="bower_components/patternfly/dist/js/patternfly.min.js"&gt;&lt;/script&gt;</code>
-          </pre>
-        </p>
-        <div class="col-sm-8 col-md-8 section-img">
-          <img src="{{ site.baseurl}}/assets/img/get-started-header.png" alt="">
-        </div>
+        <p>Before you build up web apps with npm and PatternFly, there are a few things that you have to do first:</p>
+        <ul>
+          <li>Make sure that you have node.js installed.</li>
+          <li><kbd>mkdir myProject && cd myProject</kbd></li>
+          <li>Type <kbd>npm init</kbd> to create package.json file.</li>
+        </ul>
       </div>
     </div>
   </div>
 </div>
 <div class="row">
-  <div class="col-md-12">
-    <h2>Step 4: Enjoy!</h2>
-    <div class="row">
-      <div class="col-xs-12 col-sm-12 col-md-12">
-        <p>
-          Now that you have everything configured, enjoy exploring and using PatternFly. For any questions, reach out to us on our <a href="mailto:patternfly@redhat.com">mailing list</a>, <a href="https://gitter.im/patternfly/patternfly?utm_source=share-link&utm_medium=link&utm_campaign=share-link" target="top">Gitter</a> or on <a href="https://webchat.freenode.net/" target="top">Freenode</a> at #patternfly.
-        </p>
-      </div>
+  <div class="col-xs-12 col-sm-12 col-md-12">
+    <h3>Step 2: Install PatternFly</h3>
+    <p>
+      <kbd>
+        npm install patternfly --save
+      </kbd>
+    </p>
+    <p>This will download and write to node_modules PatternFly and its dependency libraries.</p>
+  </div>
+</div>
+<div class="row">
+  <div class="col-xs-12 col-sm-12 col-md-12">
+    <h3>Step 3: Apply PatternFly in your project</h3>
+    <p>Now we can includ the necessary assets by directly specifying their path inside the node_modules folder.</p>
+  </div>
+  <div class="col-xs-12 col-sm-12 col-md-12">
+    <pre class="prettyprint">
+      <code class="language-html">{% escape_html %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.24.0/css/patternfly.min.css">
```

Comment:
Once a user installs patternfly with npm, they can consume patternfly css from their node_modules folder and stop using the CDN, right?